### PR TITLE
[CF1] Adding changelogs for casb, dlp, and gateway

### PR DESCRIPTION
**Pull request by crwaters16**

### CODE REVIEW

File: content/cloudflare-one/changelog/casb.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    title: CASB Changelog
+    description: Review recent changes to Cloudflare CASB.
+layout: changelog
+changelog_file_name: [casb]
+outputs:
+    - html
+    - rss
+---
+
+# Access Changelog
```
Comment (ranbel) [inline content/cloudflare-one/changelog/casb.md]:
```suggestion
# CASB Changelog
```

File: content/cloudflare-one/changelog/dlp.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    title: DLP Changelog
+    description: Review recent changes to Cloudflare DLP.
+layout: changelog
+changelog_file_name: [dlp]
+outputs:
+    - html
+    - rss
+---
+
+# Access Changelog
```
Comment (ranbel) [inline content/cloudflare-one/changelog/dlp.md]:
```suggestion
# DLP Changelog
```

File: content/cloudflare-one/changelog/zt-gateway.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    title: Gateway Changelog
+    description: Review recent changes to Cloudflare Gateway.
+layout: changelog
+changelog_file_name: [zt-gateway]
+outputs:
+    - html
+    - rss
+---
+
+# Access Changelog
```
Comment (ranbel) [inline content/cloudflare-one/changelog/zt-gateway.md]:
```suggestion
# Gateway Changelog
```

File: data/changelogs/dlp.yaml [hunk 4]
```diff
@@ -0,0 +1,8 @@
+---
+link: "/cloudflare-one/changelog/casb/"
```
Comment (ranbel) [inline data/changelogs/dlp.yaml]:
```suggestion
link: "/cloudflare-one/changelog/dlp/"
```

File: data/changelogs/dlp.yaml [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+productName: DLP
+entries:
+- publish_date: '2024-04-16'
+  title: Optical character recognition
+  description: |-
+    DLP can now [detect sensitive data](/cloudflare-one/policies/data-loss-prevention/dlp-profiles/advanced-settings/#optical-character-recognition-ocr) in jpeg, jpg, and pgn files. This helps companies prevent the leak of sensitive data in images, such as screenshots.
```
Comment (ranbel) [inline data/changelogs/dlp.yaml]:
```suggestion
    DLP can now [detect sensitive data](/cloudflare-one/policies/data-loss-prevention/dlp-profiles/advanced-settings/#optical-character-recognition-ocr) in jpeg, jpg, and png files. This helps companies prevent the leak of sensitive data in images, such as screenshots.
```

File: data/changelogs/zt-gateway.yaml [hunk 6]
```diff
@@ -0,0 +1,8 @@
+---
+link: "/cloudflare-one/changelog/gateway/"
```
Comment (ranbel) [inline data/changelogs/zt-gateway.yaml]:
```suggestion
link: "/cloudflare-one/changelog/zt-gateway/"
```

File: content/cloudflare-one/changelog/zt-gateway.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
+---
+pcx_content_type: changelog
+title: Gateway
+meta:
+    title: Gateway Changelog
+    description: Review recent changes to Cloudflare Gateway.
+layout: changelog
+changelog_file_name: [zt-gateway]
```
Comment (ranbel) [inline content/cloudflare-one/changelog/zt-gateway.md]: Is there a reason we call this `zt-gateway` instead of just `gateway`?

**Review by ranbel — APPROVED** [review]

File: content/cloudflare-one/changelog/zt-gateway.md (hunk 7 above)
Comment (crwaters16) [reply content/cloudflare-one/changelog/zt-gateway.md]: We have the AI gateway product also and need to distinguish between the two

**Pull request merged**